src/lib/bootloader/grub2_widgets.rb

```diff
@@ -123,50 +123,9 @@ def store
     end
 
     # Represents decision if smt is enabled
-    class CpuMitigationsWidget < CWM::ComboBox
+    class GrubCpuMitigationsWidget < CpuMitigationsWidget
```

Comment:
and btw if you do as I suggested below with single widget in generic widgets, then to avoid changes in yast2-installation you can add just alias here with simple
```
CpuMitigationsWidget = ::Bootloader::CpuMitigationsWidget
```
and it will stay backward compatible :wink:

Reply:
Yes, I have "unified" it. Good idea .....